tutorials/24_Building_Chat_Application.ipynb

```diff
@@ -0,0 +1,1953 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "TEebQqubxa4G"
+   },
+   "source": [
+    "# Tutorial: Building a Human-like Chat Application\n",
```

Comment:
How about "building a chat application with ConversationalAgent"?

Reply:
I'd like to emphasize that the chat application is human-like or conversational in the title. Also, I noticed that we have the same _"Doing something **with** something"_ structure in all tutorial titles. I lean toward updating the title for this one :)

Reply:
Ah ok, my main comment was around having the node we highlight in the title, which was the structure that Aga came up with if I remember correctly. But we can also make the conscious decision to change that if you like. If @dfokina and you both agree on this, I'm good 👍 -

Reply:
IMO it's good to have the node in the title – users will be able to find it easier and more drawn to try it out. I just looked up that plenty of our tutorials don't do that, which can be confusing for me personally.
The easiest would be _Building a Human-like Chat App with ConversationalAgent_, but that's a tad too long.

How about _Building a Conversational Chat App_? It suggest both an idea of human-like natural language, and the use of ConversationalAgent  🤔

Reply:
I'll confirm with Aga tomorrow, but I think it's okay to use "app", because we're using more informal language in haystack anyway :)